.azure-pipelines.yml

```diff
@@ -0,0 +1,89 @@
+#
+# Azure Pipelines configuration for building and testing react on Linux, Windows, and macOS.
+#
+
+trigger:
+- master
+
+jobs:
+- job: LinuxBasic
+  displayName: 'Linux: Basic Tests'
+  pool:
+    vmImage: 'ubuntu-16.04'
+  steps:
+    - template: .azure-pipelines/common/setup.yml
+    - template: .azure-pipelines/linux/basic-tests.yml
+    - template: .azure-pipelines/common/publish-error-codes.yml
+      parameters:
+        artifactName: 'error-codes-linux-basic'
+
+- job: LinuxProd
+  displayName: 'Linux: Prod Tests'
+  pool:
+    vmImage: 'ubuntu-16.04'
+  steps:
+    - template: .azure-pipelines/common/setup.yml
+    - template: .azure-pipelines/linux/prod-tests.yml
+    - template: .azure-pipelines/common/publish-error-codes.yml
+      parameters:
+        artifactName: 'error-codes-linux-prod'
+
+- job: LinuxFire
+  displayName: 'Linux: React Fire Tests'
+  pool:
+    vmImage: 'ubuntu-16.04'
+  steps:
+    - template: .azure-pipelines/common/setup.yml
+    - template: .azure-pipelines/linux/fire-tests.yml
+    - template: .azure-pipelines/common/publish-error-codes.yml
+      parameters:
+        artifactName: 'error-codes-linux-fire'    
+
+- job: LinuxArtifacts
+  displayName: 'Linux: Build and Upload Artifacts'
+  pool:
+    vmImage: 'ubuntu-16.04'
+  steps:
+    - template: .azure-pipelines/common/setup.yml
```

Comment:
I'm not crazy about "setup" since this doesn't really describes what's happening. With that said, I don't have a great alternative. Maybe `install.yml` since this is mainly about running yarn install?

Reply:
Sure, `install` works. Installing Node and other packages :)